contracts/0xcert/ERC165.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.23;
+
+/**
+ * @dev A standard for detecting smart contract interfaces. See https://goo.gl/cxQCse.
+ */
+interface ERC165 {
+
+  /**
+   * @dev Checks if the smart contract includes a specific interface.
+   * @notice This function uses less than 30,000 gas.
+   * @param _interfaceID The interface identifier, as specified in ERC-165.
+   */
+  function supportsInterface(
+    bytes4 _interfaceID
+  )
+    external
+    view
+    returns (bool);
```

Comment:
Can you update the style of these smart contracts to reflect our own? Right now they are putting a modifier on every line which is quite different how ours are written.